src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -212,7 +222,7 @@ class VanillaUmiConsensusCaller(override val readNamePrefix: String,
         consensusQuals(positionInRead) = qual
 
         // Generate the values for depth and count of errors
-        val depth  = builder.contributions
+        val depth  = contributions
```

Comment:
Maybe just move this line up to where you define `contributions` now, and use `depth` in the other spot too?